Makefile

```diff
@@ -189,8 +189,10 @@ ifeq ($(DEBUG), y)
 	OPTIMIZATIONS:=g
 endif
 
-override CFLAGS+=-O$(OPTIMIZATIONS) -Wall -Werror -ffreestanding -std=c11 -pedantic -pedantic-errors \
-	-fno-pic $(arch-cflags) $(platform-cflags) $(CPPFLAGS) $(debug_flags)
+
```

Comment:
Is it better with extra space?

Reply:
The next PR in the chain modifies those lines. It is properly formatted there.